plugin/gatsby-browser.js

```diff
@@ -40,5 +41,14 @@ export const onInitialClientRender = (_, options) => {
     }
   };
 
-  frame();
+  if (season) {
+    if (
+      new Date(now).getTime() >= new Date(season.start).getTime() &&
```

Comment:
**question:** Aren't season.start already a date?

Reply:
They are but they get passed as `string` so you can't run `getTime()` on the unless the `new Date` constructor is used

Reply:
I see!